pkg/backup/backuper.go

```diff
@@ -0,0 +1,326 @@
+package backup
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"path"
+	"time"
+
+	"github.com/mattn/go-sqlite3"
+
+	"github.com/pkg/errors"
+)
+
+// BackupFilenamePrefix is the prefix used in every backup file.
+const BackupFilenamePrefix = "tbl_backup"
+
+var (
```

Comment:
`open` and `newBackupFile` are global variable functions. This was done, so these function variables can be mocked in the tests. It will make sense when looking at the tests.